### ISSUE TITLE
columns select incorrect in MySQL source

### ISSUE THREAD

**Issue by GitHubWuYi** [mw-experts/rsql#29]
const rsqlStr = 'name=="Kill Bill",year=ge=2000';
let result;

try {
  result = rsql.RsqlToSqlConverter.getInstance().convert(rsqlStr);
} catch (e) {
  console.warn(e);
}

console.log(result);

// will output:
// ("name" = 'Kill Bill' OR "year" >= '2000')

in MySQL, "name" or "year" means strings ,not column name.
MySQL needs to be like (name = 'Kill Bill' OR year >= '2000') or (\`name\` = 'Kill Bill' OR \`year\` >= '2000').

now, my solution is: replace "${node.field}" to \`${node.field}\` in rsql-to-sql-converter.js by manually.
is there any  other  beautiful way?

thank you.

**Comment by misticwonder** [comment]
@GitHubWuYi I have added support for different syntax. It is available in package version `@mw-experts/rsql@1.6.0`.

You can have different field names wrapper, for example if you use Mysql.
Just pass a second argument to `convert` method like this:

```
const rsqlStr = 'name=="Kill Bill",year=ge=2000';
let result;

try {
  result = rsql.RsqlToSqlConverter.getInstance().convert(rsqlStr, '`');
} catch (e) {
  console.warn(e);
}

console.log(result);

// will output:
// (`name` = 'Kill Bill' OR `year` >= '2000')
```

**Comment by GitHubWuYi** [comment]
@misticwonder Thank you very much.
